AppController/lib/nginx.rb

```diff
@@ -769,12 +769,22 @@ def self.create_app_config(my_public_ip, my_private_ip, proxy_port,
 
   # Set up the folder structure and creates the configuration files necessary for nginx
   def self.initialize_config
+    nginx_pid_file = '/var/run/nginx.pid'
+    begin
+      os_version = `lsb_release -c`
+      if os_version.split.last == 'trusty'
+        nginx_pid_file = '/run/nginx.pid'
+      end
```

Comment:
Given that all modern distro have /run, and /var/run is a symlink to it, I would propose either to try /run also on precise, or to reverse the logic, and put the pid in /var/run only if it is precise, so we will be able to remove it as soon as we move away from precise.